test/verifiedTests/testrFBA/testdynamicRFBA.m

```diff
@@ -0,0 +1,53 @@
+% The COBRAToolbox: testdynamicRFBA.m
+%
+% Purpose:
+%     - testrFBA tests the dynamicRFBA function and its different outputs
+%
+% Author:
+%     - Marouen BEN GUEBILA - 31/01/2017
+ 
+ 
+% define the path to The COBRAToolbox
+pth = which('initCobraToolbox.m');
+CBTDIR = pth(1:end - (length('initCobraToolbox.m') + 1));
+ 
+cd([CBTDIR filesep 'test' filesep 'verifiedTests' filesep 'testrFBA'])
+ 
+load modelReg;
+load testDatadynamicRFBA;
+
+%Solver packages
+solverPkgs = {'tomlab_cplex'};
+%QP solvers
+QPsolverPkgs = {'tomlab_cplex'};
+
+for k =1:length(solverPkgs)
+    solverLPOK = changeCobraSolver(solverPkgs{k},'LP');
+    for j=1:length(QPsolverPkgs)%QP solvers
+        solverQPOK = changeCobraSolver(QPsolverPkgs{j},'QP');
```

Comment:
I left the lines commented for you to add them with initTest and all the tricks. 
otw the test passes.